static/js/show-task.js

```diff
@@ -39,20 +41,56 @@ function saveTask() {
     }});
 }
 
+function rememberState() {
+	// Finding how many number of rows are being displayed so that we can make it same after refresh
+        var dropdownElements = $('.page-list .dropdown-menu .dropdown-item');
+        var activeDropdownElement = $('.page-list .dropdown-menu .dropdown-item.active')[0];
+        for(var i=0; i < dropdownElements.length; i++)
+        {
+            if (dropdownElements[i] === activeDropdownElement) {
+                activeDropdownElementIndex = i;
+            }
+        }
+
+        // Finding which page is active so that we can make it active again after refresh
+        var paginationElements = $('.pagination .page-item .page-link');
+        var activeElement = $('.pagination .page-item.active .page-link')[0];
+        for(var i=0; i < paginationElements.length; i++)
+        {
+            if (paginationElements[i] === activeElement) {
+                activeElementIndex = i;
+            }
+        }
+}
+
+function restoreState() {
+      // making the number of rows to be same
+      var dropdownElements = $('.page-list .dropdown-menu .dropdown-item');
+      dropdownElements[activeDropdownElementIndex].click();
```

Comment:
I'm seeing  an exception here in the live tool with my Colombia worklist after an autorefresh:`dropdownElements[activeDropdownElementIndex] is undefined`

Is it running the latest version of this PR?

Reply:
Yes, it has the latest version only. It happened because the load function didn't complete within 2 seconds for Columbia worklist. After 2 seconds the timeout got triggered and restoreState function was called.
We need to either increase the timeout duration for this or find a better solution.

Reply:
I've fixed this issue by using Mutation Observers upon which arrive.js (https://github.com/uzairfarooq/arrive) is built, and executed the function for restoring pagination state only after pagination elements were added into the DOM.